src/v3/src/types/schema.ts

```diff
@@ -240,7 +240,7 @@ export interface UISchemaLayout extends Layout {
 
 export interface StepperLayout extends Layout {
   type: UISchemaLayoutType.STEPPER;
-  elements: Omit<UISchemaLayout, 'StepperLayout'>[];
+  elements: Omit<UISchemaLayout['elements'], 'StepperLayout'>;
```

Comment:
Is this another small throw-in change?

Reply:
lol yea, this doesn't functionally impact anything but when using types, this is the proper way to model this interface to exclude the stepper from nesting itself.